mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -606,13 +672,28 @@
     </CAbilMorph>
     <CAbilMorph id="MorphToLurker">
         <Flags index="IgnoreFacing" value="1"/>
+        <Flags index="WaitUntilStopped" value="0"/>
+        <InfoArray index="0" RandomDelayMax="0"/>
     </CAbilMorph>
     <CAbilMorph id="MorphToOverseer">
         <Flags index="IgnoreFacing" value="1"/>
         <Flags index="WaitUntilStopped" value="0"/>
     </CAbilMorph>
     <CAbilMorph id="MorphToRavager">
         <Flags index="IgnoreFacing" value="1"/>
+        <Flags index="WaitUntilStopped" value="0"/>
```

Comment:
Fixed an issue where Morph to Ravager would be canceled with a Smart command issued immediately after the Morph command.

The Rpach is not required to deaccelerate to speed 0 anymore before the morph starts. During deacceleration, other commands could cancel this morph as it was still in its preparation phase.